### ISSUE TITLE
SDAT - Writing Files | No Transform Defined

### ISSUE THREAD

**Issue by Tstarr27** [openmrslab/suspect#106]
I'm running into an issue with the suspect.io.lcmodel.write_all_files function - it's producing the following error.

```
`ValueError                                Traceback (most recent call last)
<ipython-input-6-b2c1b9d3a1ac> in <module>()
     15 }
     16 
---> 17 suspect.io.lcmodel.write_all_files(lc_model_save_loc, phased_svs, wref_data= phased_wref, params=params)

/home/sai/workspace/suspect/suspect/io/lcmodel.py in write_all_files(filename, data, wref_data, params)
    132         base_params["FILCOR"] = "{}".format(os.path.join(folder, file_root + ".CORAW"))
    133 
--> 134     save_raw(base_params["FILRAW"], data)
    135     if wref_data is not None:
    136         save_raw(base_params["FILH2O"], wref_data)

/home/sai/workspace/suspect/suspect/io/lcmodel.py in save_raw(filename, data)
     34         fout.write(" FMTDAT = '(2E15.6)'\n")
     35         # convert the volume from mm^3 to cc
---> 36         fout.write(" VOLUME = {}\n".format(data.voxel_volume() * 1e-3))
     37         fout.write(" $END\n")
     38         for point in numpy.nditer(data, order='C'):

/home/sai/workspace/suspect/suspect/mrsobjects.py in voxel_volume(self)
    186 
    187         """
--> 188         return numpy.prod(self.voxel_size)
    189 
    190     def slice_hz(self, lower_bound, upper_bound):

/home/sai/workspace/suspect/suspect/base.py in wrapper(self, *args, **kwargs)
     10     def wrapper(self, *args, **kwargs):
     11         if self.transform is None:
---> 12             raise ValueError("No transform set for {} object {}".format(type(self), self))
     13         return func(self, *args, **kwargs)
     14 

ValueError: No transform set for <class 'suspect.mrsobjects.MRSData'> object <MRSBase instance f0=127.806817MHz TE=35.0ms dt=0.5ms>`
```

I do not have any problems running write_all_files for twix and RDA files. I checked and the transform of phased_data is not defined. Can I set random numbers for that / will it affect how I process the data in LC Model?

**Comment by Tstarr27** [comment]
Additionally - is there anyway to frequency correct the SDAT data as well? Is it like working with RDA file that you can't do frequency correction on it?

**Comment by Tstarr27** [comment]
I was able to correct the data doing the following:
```
print(phased_svs.transform)
phased_svs.transform = [[1,1,1,1],[1,1,1,1],[1,1,1,1],[1,1,1,1]]
print(phased_svs.transform)
phased_wref.transform = [[1,1,1,1],[1,1,1,1],[1,1,1,1],[1,1,1,1]]
```

Will that affect the LCMODEL output?

**Comment by bennyrowland** [comment]
This issue has been split into two daughter issues, one of which is resolved (#107) and the other still open (#108), but I am going to mark this one as closed and leave future development to happen on the other issue.
